### ISSUE TITLE
[Bug]: Coalition Ranger Ship Stuff

### ISSUE THREAD

**Issue by lavillastrangiato** [Aurorastation/Aurora.3#16558]
### Checks

- [X] I have searched the bug with a few keywords, and I confirm this bug was not yet reported.
- [X] The round I am reporting the bug from, or I am going to talk about, has already ended.

### Location

Horizon, Away Site, Third party ship

### Describe the issue

TGUI stuff:
1. TGUI does not have a Rest verb.
2. TGUI also does not have a Set Flavour Text verb.
3. TGUI does not have a Show Held Object verb. 
4. **You still can't vote transfer if you're not a mod.**
![image](https://github.com/Aurorastation/Aurora.3/assets/63124803/d2a7d4cd-f1b7-4808-999c-cb2b9208d25f)

Coalition Ranger ship stuff:
1. There are no sinks in the kitchen or the surgery bay.
2. There are no NanoMeds in medical (even the Kataphract ship has them!).
3. Owing to the position of the janitorial cart, and how moving the cart means you move it like a wheelchair (where your arrow keys control the object itself), the janitorial cart cannot be moved from the closet.
4. The bathroom windoor cannot be opened by anyone and has to be broken open.
5. There should probably be off-worlder bracers for off-worlder humans, as Scarabs are part of the Coalition.
6. I would like a hydroponics bay but I'd also accept an extra produce box. This isn't a missing thing it's just my opinion.

### How to reproduce

For the crew transfer:
1. Click Vote as a player.
2. Bring up menu to select restart, vote crew transfer, etc.
3. All the options are greyed out.
4. If you hover over the Crew Transfer option, a pop-up claims the transfer is not selectable on Code Red.

### Round ID

cow-bbiA

**Comment by NonQueueingMatt** [comment]
Can't replicate the missing verbs, need more info on how you spawned in

**Comment by lavillastrangiato** [comment]
I decided to try and replicate this by spawning as an offship. I clicked to spawn in as a human on the Space Bar, as a Station Visitor. Below is my UI IC tab.

![image](https://github.com/Aurorastation/Aurora.3/assets/63124803/d8d549b9-7ce8-4694-a442-a0700063f849)

Shortly after I spawned in, the customization screen that pops up when you spawn in as an away site vanished. I could not find it while opening F11 on Windows to look through my open windows. I browsed through the Preferences tab and decided to hit "Refresh TGUI".

This restores all the missing verbs.
![image](https://github.com/Aurorastation/Aurora.3/assets/63124803/bd77b2be-7382-4f07-86f9-e209cc0490f5)

However, instead of restoring my customization screen, I was met with a big blank box in the middle of my UI. This was unable to be moved or closed out of. 

![image](https://github.com/Aurorastation/Aurora.3/assets/63124803/3e822661-3e3c-4568-9819-43005526ea97)

Reconnecting in the Client options did not fix this, so I relogged to see if it would get rid of it.

This restored the previous view of my screen, so I decided to see if I could restore the missing pop-up by refreshing TGUI again. It did not work.
